### DISCUSSION TITLE
Errors using wasm-server-runner to test eframe web version

### DISCUSSION THREAD

**Discussion by Kulasko** [emilk/egui#2081] [Q&A]
Hello, I have witnessed some erroneous behaviour, but I am not sure the error isn't on my side or my configuration is supported, so I am a bit reluctant to open an issue.

I tried to model the process like I successfully did in Bevy, so no wasm-bindgen attribute, only using the normal main function. Also, I injected the canvas into the page using web-sys.

Using trunk with only one very simple html file works perfectly fine.
Using wasm-server-runner, the GUI shows up and hover effects work fine. However, on the first click on it, it vanishes.

Chrome reports:
`GL_INVALID_FRAMEBUFFER_OPERATION: Framebuffer is incomplete: Attachment has zero size.`

Firefox reports:
`WebGL warning: drawElementsInstanced: TEXTURE_2D at unit 0 is incomplete: The dimensions of 'level_base' are not all positive.`
`WebGL warning: clear: Framebuffer not complete. (status: 0x8cd6) COLOR_ATTACHMENT0: Attachment has no width or height.`
`WebGL warning: clear: Framebuffer must be complete. `

I know that the template only mentions trunk, but it normally shouldn't be dependent on it, does it?
Do I need to apply some special settings, perhaps?

**Comment by fakeshadow** [comment]
I had a similar issue the other day and the reason turned out that I enabled strip in release profile and it built broken release wasm somehow.

**Reply by Kulasko** [reply]
That is also a valid concern for which I hope it can be fixed. I've had that problem when using Bevy, so I am glad it solved the issue for you. However, it sadly isn't the issue in my instance.
